tools/src/main/scala/caliban/tools/SchemaWriter.scala

```diff
@@ -1,9 +1,12 @@
 package caliban.tools
 
+import caliban.parsing.adt.Definition.TypeSystemDefinition.TypeDefinition
 import caliban.parsing.adt.Definition.TypeSystemDefinition.TypeDefinition._
 import caliban.parsing.adt.Type.{ ListType, NamedType }
 import caliban.parsing.adt.{ Document, Type }
 
+import scala.collection.compat.toTraversableLikeExtensionMethods
```

Comment:
It looks like this import only works on 2.12. Maybe import `._` if it's not needed for 2.13 and 3?

Reply:
👍